client/src/components/CreatePostPage/UploadNotes.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react'
+import axios from 'axios';
+
+function UploadNotes() {
+    const [selectedFile, setSelectedFile] = useState(null);
+
+    const handleFileInputChange = (event) => {
+        setSelectedFile(event.target.files[0]);
+        console.log(event.target.files[0]);
+    }
+
+    const handleFileUpload = () => {
+        const formData = new FormData();
+        formData.append('pdf', selectedFile);
+        axios.post('http://localhost:4000/upload', formData)
```

Comment:
You can use env files here to make the URL's dynamic as later you need to change it only in one file i.e. your .env file. To know more -> https://blog.bitsrc.io/a-gentle-introduction-to-env-files-9ad424cc5ff4#:~:text=env%20files%20contain%20credentials%20in,env%20files%20for%20each%20environment.